tools/circle_plus_gen/lib/train_info.py

```diff
@@ -0,0 +1,97 @@
+import json
+
+from schema import circle_traininfo_generated as ctr_gen
+'''
+Wrappers of the auto generated classes in circle_traininfo_generated
+
+Wrapper classes provides additional interfaces(e.g. initializer) to the auto generated classes.
+'''
+
+# Optimizers
+
+
+class SGD(ctr_gen.SGDOptionsT):
+    name = ['sgd', 'stocasticgradientdescent']
+
+
+class Adam(ctr_gen.AdamOptionsT):
+    name = ['adam']
+
+
+# Loss
+
+
+class SparseCategoricalCrossEntropy(ctr_gen.SparseCategoricalCrossentropyOptionsT):
+    name = [
+        'sparse categorical crossentropy', 'sparsecategoricalcrossentropy', 'sparsecce'
+    ]
+
+
+class CategoricalCrossEntropy(ctr_gen.CategoricalCrossentropyOptionsT):
+    name = ['categorical crossentropy', 'categoricalcrossentropy', 'cce']
+
+
+class MeanSqauaredError(ctr_gen.MeanSquaredErrorOptionsT):
+    name = ['mean squared error', 'mse']
```

Comment:
These names will be used for parsing json file. (ref: https://github.com/Samsung/ONE/pull/12975/files#r1595499773)